python/idsse_common/test/test_grid_proj.py

```diff
@@ -1,13 +1,15 @@
-"""Test suite for grid_proj.py"""
-# --------------------------------------------------------------------------------
+"""Test suite for grid_geo.py"""
```

Comment:
Did you mean to rename grid_proj.py to grid_geo.py? Not sure if the module was actually renamed.